Dockerfile.spark

```diff
@@ -16,6 +16,8 @@ ENV LIVY_BUILD_PATH /apps/build/livy
 ENV PYSPARK_PYTHON python2.7
 ENV PYSPARK3_PYTHON python3.4
 
+RUN pip install --upgrade setuptools
```

Comment:
Hey, thanks for your submission! This seems to be very similar to #411. Do you mind updating Dockerfile.jupyter too and adding pip install to both. I can merge whichever does it first :) 

If you want to wait for the other one, feel free to close this too.

Thanks!